server/src/apub/community.rs

```diff
@@ -153,31 +127,33 @@ pub async fn get_apub_community_followers(
   Ok(create_apub_response(&collection))
 }
 
-/// Returns an UnorderedCollection with the latest posts from the community.
-pub async fn get_apub_community_outbox(
-  info: Path<CommunityQuery>,
-  db: web::Data<Pool<ConnectionManager<PgConnection>>>,
-) -> Result<HttpResponse<Body>, Error> {
-  let community = Community::read_from_name(&&db.get()?, info.community_name.to_owned())?;
-
-  let conn = establish_unpooled_connection();
-  //As we are an object, we validated that the community id was valid
-  let community_posts: Vec<Post> = Post::list_for_community(&conn, community.id)?;
-
-  let mut collection = OrderedCollection::default();
-  let oprops: &mut ObjectProperties = collection.as_mut();
-  oprops
-    .set_context_xsd_any_uri(context())?
-    .set_id(community.actor_id)?;
-  collection
-    .collection_props
-    .set_many_items_base_boxes(
-      community_posts
-        .iter()
-        .map(|c| c.as_page(&conn).unwrap())
-        .collect(),
-    )?
-    .set_total_items(community_posts.len() as u64)?;
-
-  Ok(create_apub_response(&collection))
-}
+// TODO should not be doing this
+// Returns an UnorderedCollection with the latest posts from the community.
+//pub async fn get_apub_community_outbox(
+//  info: Path<CommunityQuery>,
+//  db: DbPoolParam,
```

Comment:
I don't know how useful the community outbox will eventually be, it will be potentially thousands of different actions that get inserted into that `activity` table, filtered by community.

Reply:
exposing outboxes for communities is probably a good idea for backfilling instances that connect after the community has been around for a while (backfilling on an as-needed basis, of course)

Reply:
Ya. We asked over at https://socialhub.activitypub.rocks about history, and a lot of even the big ones completely ignore history, IE a users posts before they became visible to your instance.

I could definitely see the value in pulling a community's whole history, but if so we'll add it much later on.